doc/man1/req.pod

```diff
@@ -242,7 +242,6 @@ days to certify the certificate for. The default is 30 days.
 
 serial number to use when outputting a self signed certificate. This
 may be specified as a decimal value or a hex value if preceded by B<0x>.
-It is possible to use negative serial numbers but this is not recommended.
```

Comment:
Ah....does that mean it was never possible and the documentation was just plain wrong? Or that it was once possible but we changed things so that it isn't?